.github/workflows/release.yml

```diff
@@ -809,6 +837,17 @@ jobs:
         rm -f bundles/*.sha256
         echo "::endgroup::"
 
+    - name: Sign installer
+      if: needs.source.outputs.is_tag == 'true'
+      shell: bash
+      env:
+        WIN_CODESIGN_CN: ${{ secrets.WIN_CERTIFICATE_COMMON_NAME }}
+      # If this is run on a fork, there may not be a certificate set up - continue in this case
+      continue-on-error: true
+      run: |
+        cd ${GITHUB_WORKSPACE}/build/bundles
+        ../../os/windows/sign.bat *.exe "${WIN_CODESIGN_CN}"
```

Comment:
Anything we can do about `*.exe` ? Being a bit more specific here might help? (honest question, no is a valid answer, as always)

Reply:
If we assume cmake >=3.19 (GHA windows runners have 3.20.2), maybe we could try to use `CPACK_POST_BUILD_SCRIPTS`

Reply:
CPACK_POST_BUILD_SCRIPTS was my first port of call, but actually getting the executable name/build directory out of it seemed to be problematic. If somebody can get it working later, it's easy enough to swap out.

TrueBrain: Probably, yes, but that involves more effort (which honestly I didn't have time for) and the possibility for errors to creep in (e.g., if somebody changes the naming convention later but misses this bit), and in practice there are going to be no other .exe files in that folder anyway - so I don't think it's really an issue.